src/gui/mainwindow.cpp

```diff
@@ -1095,15 +1104,31 @@ void MainWindow::dropEvent(QDropEvent *event)
         files = event->mimeData()->text().split('\n');
     }
 
-    // Add file to download list
+    // differentiate ".torrent" files and others
+    QStringList torrentFiles, otherFiles;
```

Comment:
it's not performance critical and the current ways are easier to read IMO.
I will keep the current code.